applications/LinearSolversApplication/CMakeLists.txt

```diff
@@ -69,6 +69,12 @@ if( USE_EIGEN_FEAST MATCHES ON )
     add_definitions(-D__MKL_SOLVERS_EE_H)
 endif()
 
+if (USE_EIGEN_SUITESPARSE)

```

Comment:
who defines this?

Reply:
The user. It's intended to be used in the same manner as `USE_MKL`.

```bash
cmake ...
      -DUSE_EIGEN_SUITESPARSE:BOOL=ON \
      ...
```

Take a look at the readme changes, I explained it there.

Reply:
Then I suggest to make it a cmake option